packages/python-packages/doc-warden/README.md

```diff
@@ -178,6 +189,21 @@ The first, `known_presence_issues`, tells warden that a presence failure detecte
 
 The `known_content_issues` parameter functions _identically_ to the `known_presence_issues` check. If a readme is listed as "already known" to have failures, the entire CI build will not be crashed by Warden.
 
+##### `package_indexing_exclusion_list` and `package_indexing_traversal_stops` Configuration
+Indexing packages is often done as part of nightly (or triggered) automation. With this being the case, sometimes `warden` may detect a PackageId that users wish to omit from the generated `packages.md` file. The Azure SDK team leverages 
+the `package_indexing_exclusion_list` array members to enable just this sort of scenario.
+
+`package_indexing_traversal_stops` is used during parse of .NET language repos _only_. This is due to how the discovery logic for readme and changelog is implemented for .NET projects. Specifically, readmes for a .csproj are often a couple directories up from their parent .csproj location!
+
+For .net, `warden` will traverse **up** one directory at a time, looking for the readme and changelog files in each traversed directory. `warden` will continue to traverse until...
+
+1. It discovers a folder with a `.sln` within it
+2. It encounters a folder that exactly matches one present in `package_indexing_traversal_stops`
+
+Note that `warden` will not even execute an index against a .NET repo _unless the traversal stops are set_. 
```

Comment:
Why not default to stopping at the root of the repo?

Reply:
[I do!](https://github.com/Azure/azure-sdk-tools/blob/3fa710ea44d4c0af980b95b1b6ffd79f3cff809d/packages/python-packages/doc-warden/warden/index_packages.py#L228)

I just make it so that a user can't accidentally forget about this.